File: app/src/main/res/drawable-v24/툴/새 폴더/SplashActivity.java
package com.example.wanna_bet70;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;

public class    SplashActivity extends Activity {//로딩창 구현
    protected  void onCreate(Bundle saveInstanceState) {
        super.onCreate(saveInstanceState);


        try{
            Thread.sleep(4000);//4초 쉬고
            Intent intent = new Intent(this, LoginActivity.class);
            startActivity(intent);
            finish();

        } catch (InterruptedException e) {
            e.printStackTrace();
        }


       // startActivity(new Intent(this,LoginActivity.class));//로그인창 오픈
       // finish();

    }



}

File: app/src/main/java/com/avon/remindfeedback/SplashActivity.kt
package com.avon.remindfeedback

import android.animation.Animator
import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import com.airbnb.lottie.LottieAnimationView
import com.avon.remindfeedback.Login.LoginActivity

class SplashActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)
        var intent= Intent(this, LoginActivity::class.java)
        val animationView = findViewById<LottieAnimationView>(R.id.animationView)

        animationView.setAnimation("2233-downloading-book.json")
        animationView.loop(false)
        animationView.playAnimation()
        animationView.addAnimatorListener(object : Animator.AnimatorListener {
            override fun onAnimationStart(animation: Animator) {
                Log.e("asd", "onAnimationStart")
            }

            override fun onAnimationEnd(animation: Animator) {
                startActivity(intent)
                finish()
            }

            override fun onAnimationCancel(animation: Animator) {
            }

            override fun onAnimationRepeat(animation: Animator) {

            }
        })


    }
}
